README.md

```diff
@@ -57,7 +59,7 @@ resources:
   source:
     uri: https://github.com/cloudfoundry-community/simple-go-web-app.git
 
-- name: resource-deploy-web-app
```

Comment:
Don't really like this rename; before it was clear that 1 resource corresponds to 1 app, whereas this makes it seem like you should always `put` to the `pws` resource to push all apps.